Vizzuality/half-earth-v3
# Fix/nrc-layers

## Use marine + land borders on NRC landing
### Description
To make clear the selection on the marine layers I changed the outline of the NRC from land to land + marine
Also, updated the name of the EZZ layer to explicitly say that is land + marine

Also, fixed the AOI detailed view chart sentence that has to show when we have only the first 25% of the SPS selected.

### Testing instructions
Go to NRC landing and hover over the countries, you should see the land + marine outline. Check in mobile too
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-691?atlOrigin=eyJpIjoiODgyOWQ3ZjVmMzQ2NGVkYTg5MzNjYTNkNTNjYjI0NTIiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-683?atlOrigin=eyJpIjoiNzY3MmM1ZTAxMWFkNGVkZTllZjNlM2NjNjE5YTgzZmEiLCJwIjoiaiJ9
